internal/watcher/watcher.go

```diff
@@ -0,0 +1,387 @@
+package watcher
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/kardianos/service"
+	"github.com/ubuntu/adsys/internal/decorate"
+	log "github.com/ubuntu/adsys/internal/grpc/logstreamer"
+	"github.com/ubuntu/adsys/internal/i18n"
+	"gopkg.in/ini.v1"
+)
+
+const (
+	gptFileName = "gpt.ini"
```

Comment:
Do you mind checking the case of this file on a real AD? I’m still pretty sure this was `GPT.ini`.

Reply:
True, but as we generate those files, it’s better to be in line with the case Microsoft is using when generating GPOs